nautobot/core/api/views.py

```diff
@@ -611,13 +613,27 @@ def get(self, request, *args, **kwargs):
 
         # Add additional data so drf-spectacular will use the Token keyword in authorization header.
         response.data["schema_auth_names"] = ["tokenAuth"]
+
+        # Pass `Titles` instance to render title from this view rather than original from Swagger View
+        response.data["view_titles"] = self.view_titles
+        del response.data["title"]
```

Comment:
Is this easier than just setting `response.data["title"] = "API Documentation"`?